scripts/zkai_attention_kv_model_faithful_quantized_attention_bridge_gate.py

```
@@ -0,0 +1,636 @@
+    size = path.stat().st_size
+    if size > max_bytes:
+        raise ModelFaithfulQuantizedAttentionBridgeGateError(f"{label} exceeds byte bound")
+    value = json.loads(path.read_text(encoding="utf-8"))
+    if not isinstance(value, dict):
+        raise ModelFaithfulQuantizedAttentionBridgeGateError(f"{label} must be an object")
+    return value
+
+
+def prepare_output_path(path: pathlib.Path) -> pathlib.Path:
+    if path.exists() and path.is_symlink():
+        raise ModelFaithfulQuantizedAttentionBridgeGateError("output path must not be a symlink")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)